miio/tests/dummies.py

```diff
@@ -66,6 +66,15 @@ def __init__(self, *args, **kwargs):
     def get_properties_for_mapping(self):
         return self.state
 
+    def get_properties(self, properties):
+        keys = [p["did"] for p in properties]
```

Comment:
Considering this modifies the common dummies, could you add a simple docstring describing what's its usecase?